### DISCUSSION TITLE
perfomance React js

### DISCUSSION THREAD

**Discussion by nguyenduong1803** [community/community#49667] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Are there ways for senior developers to optimize their websites?

**Comment by maahaMaya** [comment, marked as answer]
Yes 
1. Yes angular material library
2. use React redux Saga :: for pass data in component and call the Api 
3. `https://www.youtube.com/watch?v=sc5EsyjOLj8`

**Reply by nguyenduong1803** [reply]
thanks for the answer. but i think redux toolkit looks better.

**Comment by Mroczek1992** [comment]
Hhh
